README.md

```diff
@@ -27,6 +27,14 @@ This repository contains the orchestra configuration for rev.ng.
   orc components
   ```
 
+### Virtualenvs
+
+We also partially support installing orchestra inside a virtualenv.
+
+You will have to install the system dependencies on your own (manually inspect the 
+`./.orchestra/ci/install-dependencies.sh` script), but you can use 
+`./.orchestra/ci/install-python-dependencies.sh` to install the Python dependencies inside the virtualenv.
+
```

Comment:
Can't we simply add an optional step before `Install dependencies (currently Ubuntu-only):` where we say to create and enter a virtualenv.

Wouldn't that work?

Reply:
> only if the user doesn't run the script as sudo

Why? And why would the user run orc as root in a venv? :P

> I'd still mention the python script at least

What Python script? `install-python-dependencies.sh`?
My idea is that I don't want people to manually look into `install-dependencies.sh` if we can easily extend our scripts to support the virtualenv use case.

Reply:
Updated, please check again 🙃

---

> Why? And why would the user run orc as root in a venv? :P

I was thinking about the install dependencies script since the system deps part needs root. But indeed `python --user` would still break with root.

> What Python script? install-python-dependencies.sh?
>  My idea is that I don't want people to manually look into `install-dependencies.sh`

[cries in archlinux]